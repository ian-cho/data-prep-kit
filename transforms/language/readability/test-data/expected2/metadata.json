{
    "pipeline": "pipeline_id",
    "job details": {
        "job category": "preprocessing",
        "job name": "readability",
        "job type": "pure python",
        "job id": "job_id",
        "start_time": "2024-10-03 19:04:11",
        "end_time": "2024-10-03 19:04:12",
        "status": "success"
    },
    "code": {
        "github": "github",
        "commit_hash": "12345",
        "path": "path"
    },
    "job_input_params": {
        "contents_column_name": "contents",
        "curriculum": 0,
        "checkpointing": false,
        "max_files": -1,
        "random_samples": -1,
        "files_to_use": [".parquet"],
        "num_processors": 0
    },
    "job_output_stats": {
        "source_files": 1,
        "source_size": 14884,
        "result_files": 1,
        "result_size": 24744,
        "processing_time": 0.206,
        "nrows": 2,
        "source_doc_count": 2,
        "result_doc_count": 2
    },
    "source": {
        "name": "/Users/touma/data-prep-kit-inner/transforms/language/readability/python/test-data/input",
        "type": "path"
    },
    "target": {
        "name": "/Users/touma/data-prep-kit-inner/transforms/language/readability/python/test-data/output",
        "type": "path"
    }
}
