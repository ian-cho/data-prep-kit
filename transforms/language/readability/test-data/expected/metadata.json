{
    "pipeline": "pipeline_id",
    "job details": {
        "job category": "preprocessing",
        "job name": "readability",
        "job type": "pure python",
        "job id": "job_id",
        "start_time": "2024-10-03 17:57:33",
        "end_time": "2024-10-03 17:57:33",
        "status": "success"
    },
    "code": {
        "github": "github",
        "commit_hash": "12345",
        "path": "path"
    },
    "job_input_params": {
        "contents_column_name": "contents",
        "curriculum": true,
        "checkpointing": false,
        "max_files": -1,
        "random_samples": -1,
        "files_to_use": [".parquet"],
        "num_processors": 0
    },
    "job_output_stats": {
        "source_files": 1,
        "source_size": 14884,
        "result_files": 1,
        "result_size": 17206,
        "processing_time": 0.171,
        "nrows": 2,
        "source_doc_count": 2,
        "result_doc_count": 2
    },
    "source": {
        "name": "/Users/touma/data-prep-kit-inner/transforms/language/readability/python/test-data/input",
        "type": "path"
    },
    "target": {
        "name": "/Users/touma/data-prep-kit-inner/transforms/language/readability/python/test-data/output",
        "type": "path"
    }
}
